src/LearnMe.Core/Services/Calendar/Utils/GoogleAPIconnection.cs

```diff
@@ -29,10 +29,10 @@ public UserCredential GetToken()
 
             UserCredential credential;
 
-            using var stream = new FileStream("Controllers\\Libraries\\CalendarController\\Utils\\credentials.json", FileMode.Open, FileAccess.Read);
+            using var stream = new FileStream("..\\LearnMe.Core\\Services\\Calendar\\Utils\\Credentials\\credentials.json", FileMode.Open, FileAccess.Read);
```

Comment:
normalnie się tego tak nie robi ale powiedzmy ze dopuszczalna sprawa jak tego nie będziecie nigdzie wystawiać

Reply:
Co masz na myśli przez "nie będziecie tego nigdzie wystawiać"? Zamysł jest taki że ludzie będą tego używać online

Reply:
https://cshark.com/blog/how-to-store-sensitive-data-of-your-application-in-net-core-app/

https://azure.microsoft.com/en-us/services/key-vault/

https://docs.microsoft.com/en-us/azure/key-vault/general/basic-concepts

Ogólnie już się nie robi tak, że trzymacie takie dane na serwerze. Jakbyście wystawiali apkę na windowsie i IIS no to powiedzmy ze spoko bo macie VM do któego się łączycie i podmieniacie te credentiale. Obecnie jednak te podejście jest bez sensu bo mamy multiplatformowy .net core i konteneryzacje. W tym drugim podejściu wygodniejsze jest trzymanie sekretów w chmurze a kod pchany w obraz dokerowy i bez problemu wystawiany do świata skądkolwiek